checkin/views.py

```diff
@@ -143,6 +95,12 @@ def get_context_data(self, **kwargs):
             profile = Profile.objects.get(user=self.request.user)
         else:
             profile = Profile.objects.create(user=self.request.user)
+
+        # Connect card number from course registration to user
+        registration = Printer3DCourse.objects.filter(user__username=self.request.user.username)
+        if registration.exists():
+            Card.update_or_create(self.request.user, "EM " + str(registration.first().card_number))
```

Comment:
I think we should not add "EM " to all the card numbers. It is much better if we just prepend it in the interface when we feel the need to show it. Until now, we have only collected the numbers and not prepended them with "EM".